main.go

```diff
@@ -1,7 +1,158 @@
 package main
 
-import "fmt"
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"math/rand"
+	"net/http"
+	"strconv"
 
+	"github.com/gorilla/mux"
+)
+
+// Certificate Struct
+type Certificate struct {
+	ID        string    `json:"id"`
+	Title     string    `json:"title"`
+	CreatedAt string    `json:"createdAt"` //should be type date?
```

Comment:
Yes https://golang.org/pkg/time/

Reply:
We can map to data type time using marshals like this
https://ukiahsmith.com/blog/go-marshal-and-unmarshal-json-with-time-and-url-data/